README.md

```diff
@@ -12,6 +12,175 @@ This is a Scala/Play frontend to allow Limited Companies to provide their inform
 3. Run the frontend locally using
 `sbt 'run 9718 -Dapplication.router=testOnlyDoNotUseInAppConf.Routes'`
 
+### End-Points
+#### POST /journey
+
+---
+Creates a new journey, storing the journeyConfig against the journeyId
+##### Request:
+No body is required for this request
+
+##### Response:
+Status: **Created(201)**
+
+Example Response body: 
+
+```
+{“journeyStartUrl” : "/testUrl"}
+```
+
+#### GET /journey/:journeyId
+
+---
+Retrieves all the journey data that is stored against a specific journeyID
+##### Request:
+A valid journeyId must be sent in the URI
+
+##### Response:
+Status:
+
+| Expected Response                       | Reason  
+|-----------------------------------------|------------------------------
+| ```OK(200)```                           |  ```JourneyId exists```       
+| ```NOT_FOUND(404)```                    | ```JourneyId doesn't exist```
+
+Example response body:
+```
+{"companyProfile":
+    {"companyName":"TestCompanyLtd”,
+    “companyNumber":"01234567",
+    "dateOfIncorporation":"2020-01-01"},
+"ctutr":"1234567890",
+"identifiersMatch":true,
+"businessVerification":
+    {"verificationStatus":"PASS"},
+"registration":
+    {"registrationStatus":"REGISTERED",
+    "registeredBusinessPartnerId":"X00000123456789"}
+}
+```
+### Test End-Points
+#### GET test-only/feature-switches
+
+---
+Shows all feature switches:
+1. Incorporated Entity Identification Frontend
+    
+    - Companies House API stub
+    - Business Verification Stub
+2. Incorporated Entity Identification
+    
+    - Get CT reference stub
+    - DES stub
+    
+#### GET/POST test-only/create-journey
+
+---
+Mocks retrieving and storing the journey config required by the calling service to pass to this service
```

Comment:
This is a test entry point which simulates a service making the initial call to setup a jounrey